compiler-core/src/language_server/engine.rs

```diff
@@ -360,7 +359,12 @@ where
                 if !type_.public {
                     continue;
                 }
-                completions.push(type_completion(Some(&alias), name, type_));
+                let module = match import.clone().as_name {
```

Comment:
This is cloning the entire import, this should take the import by reference instead.
```suggestion
                let module = match import.as_name.as_ref() {
```